cni/network/plugin/main.go

```diff
@@ -173,7 +174,7 @@ func main() {
 
 	cniReport.GetReport(pluginName, version, ipamQueryURL)
 
-	netPlugin, err := network.NewPlugin(name, &config)
+	netPlugin, err := network.NewPlugin(name, &config, &nnsgrpcclient.NnsGrpcClient{})
```

Comment:
Just a thought on package naming, it's very common in Go to have packages reflect domain, such that types within can have succinct names, without losing context from a callers perspective. You could have an `nns` package, and then types such as `nns.Client`

Reply:
Moved it outside of plugin and renamed the package.